include/ServerMainComponent.hpp

```diff
@@ -95,6 +95,9 @@ class ServerMainComponent : public juce::Component
 	/// @returns True if the client's active object pool was deactivated and removed from volatile storage, otherwise false.
 	bool delete_object_pool(isobus::NAME clientNAME) override;
 
+	uint8_t get_user_layout_datamask_bg_color() const override;
+	uint8_t get_user_layout_softkeymask_bg_color() const override;
```

Comment:
Well, sorta, please also change these here and in the cpp file to 
```suggestion
	std::uint8_t get_user_layout_datamask_bg_color() const override;
	std::uint8_t get_user_layout_softkeymask_bg_color() const override;
```